ait/dsn/plugins/AOS_FEC_Check.py

```diff
@@ -1,17 +1,21 @@
 from ait.core.server.plugins import Plugin
 from ait.core import log
-from ait.dsn.sle.frames import AOSTransFrame, AOSDataFieldType
+from ait.dsn.sle.frames import AOSTransFrame
 from binascii import crc_hqx
 from dataclasses import dataclass
 import ait.dsn.plugins.Graffiti as Graffiti
 
 
+STRICT = False
+
+
 @dataclass
 class TaggedFrame:
     frame: bytearray
     vcid: int = None
-    counter: int = None
+    absolute_counter: int = None
```

Comment:
does this count every single TaggedFrame ever made?

Reply:
Yeah.
I currently can't think of a reason to use it in production though.

I just use it to find out how long we were able to process before something bad happened.